ififuncs.py

```diff
@@ -899,7 +899,16 @@ def get_script_version(scriptname):
     current_dir = os.getcwd()
     home = os.path.expanduser("~/")
     os.chdir(home)
-    if os.path.isdir('ifigit/ifiscripts'):
+    pypi = subprocess.check_output([
```

Comment:
Also, if pip or pip3 is not installed, and a user only downloads the git repo, this check will result in a fatal error. You should wrap it in a try/except loop in case pip isn't installed, or some other method of accounting for pip not being installed

Reply:
fixed:
```
pypi = ''
try:
    pypi = subprocess.check_output(['pip', 'show', 'ifiscripts'])
    pypi = pypi.decode()
except Exception as e:
    print(e)
if pypi:
......
```